docs/deployment/private-ddn/create-a-project-on-a-data-plane.mdx

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 3
+sidebar_label: Create a project
+description: "Learn how to create a project on a private Data Plane."
+keywords:
+  - hasura ddn
+  - private ddn
+  - dedicated vpc
+  - enterprise ddn
+seoFrontMatterUpdated: false
+---
+
+import Thumbnail from "@site/src/components/Thumbnail";
+import Prereqs from "@site/docs/_prereqs.mdx";
+
+# How to Create a Project on a Data Plane
+
+## Introduction
+
+This guide will walk you through the steps for creating a project on a Private Hasura DDN Data Plane. Before proceeding,
+you'll first need to be [invited](/deployment/private-ddn/data-plane-collaboration.mdx) to a Data Plane.
+
+<Prereqs />
+
+## Step 1. Initialize a local project
+
+```sh title="Being by authenticating your CLI seession:"
+ddn auth login
+```
+
+A browser window will open and prompt you for your login credentials. After successfully logging in, you'll be able to
+return to the CLI.
+
+```sh title="Next, creat a new local project:"
+ddn supergraph init <project_name> && cd <project_name>
+```
+
+The DDN CLI will scaffold out all the necessary files and directories for your project.
+
+## Step 2. Retrieve your Data Plane ID
+
+You can find all the Data Planes you have access to on this page: https://console.hasura.io/data-plane
+
+## Step 3. Create a project on your Data Plane
+
+You'll create a new project using the DDN CLI and by passing two values: the `data-plane-id` (retrieved above) and the
+`plan` which is a string value of either:
+
+- `base`
+- `advanced`
+
+```sh title="Create a new project using your Data Plane ID and plan choice:"
+ddn project create --data-plane-id <data-plane-id> --plan <base|advanced>
```

Comment:
```suggestion
ddn project init --data-plane-id <data-plane-id> --plan <ddn_base|ddn_advanced>
```

Reply:
We can add a note that the user can change the plan later by reaching out to support.